### PR TITLE
Update Sql Migration extension strings

### PR DESCRIPTION
**Pull request by rchlkm**
This PR updates the localized string for SQL migration extension based on feedback from content reviewer.

### CODE REVIEW

File: extensions/sql-migration/src/constants/strings.ts [hunk 1]
```diff
@@ -120,10 +119,10 @@ export const DATABASE_BACKUP_FILE_SHARE_SUBSCRIPTION_LABEL = localize('sql.migra
 export const DATABASE_BACKUP_FILE_SHARE_STORAGE_ACCOUNT_LABEL = localize('sql.migration.file.share.storage.account.label', "Select the storage account that contains the file share.");
 export const DATABASE_BACKUP_FILE_SHARE_LABEL = localize('sql.migration.file.share.label', "Select the file share that contains the backup files.");
 export const DATABASE_BACKUP_FILE_SHARE_PLACEHOLDER = localize('sql.migration.file.share.placeholder', "Select share");
-export const DATABASE_BACKUP_MIGRATION_MODE_LABEL = localize('sql.migration.database.migration.mode.label', "Migration mode");
-export const DATABASE_BACKUP_MIGRATION_MODE_DESCRIPTION = localize('sql.migration.database.migration.mode.description', "Choose from the following migration modes to migrate to your Azure SQL target based on your downtime requirements.");
+export const DATABASE_BACKUP_MIGRATION_MODE_LABEL = localize('sql.migration.database.migration.mode.label', "Migration Mode");
+export const DATABASE_BACKUP_MIGRATION_MODE_DESCRIPTION = localize('sql.migration.database.migration.mode.description', "To migrate to the Azure SQL target, choose a migration mode based on your downtime requirements.");
```
Comment (brian-harris) [inline extensions/sql-migration/src/constants/strings.ts:123]: should be sentenced cased "Migration mode"

File: extensions/sql-migration/src/constants/strings.ts [hunk 2]
```diff
@@ -180,32 +179,32 @@ export const SELECT_BLOB_CONTAINER = localize('sql.migration.blob.container.sele
 
 // integration runtime page
 export const IR_PAGE_TITLE = localize('sql.migration.ir.page.title', "Azure Database Migration Service");
-export const IR_PAGE_DESCRIPTION = localize('sql.migration.ir.page.description', "Azure Database Migration Service (DMS) orchestrates database migration activities and tracks their progress. You can select an existing DMS for Azure SQL target if you have created one previously or create a new one below.");
+export const IR_PAGE_DESCRIPTION = localize('sql.migration.ir.page.description', "Azure Database Migration Service (DMS) orchestrates database migration activities and tracks their progress. You can select an existing DMS as an Azure SQL target if you have created one previously, or create a new one below.");
 export const IR_PAGE_NOTE = localize('sql.migration.ir.page.note', "Note: DMS will run in your Azure subscription in the chosen resource group and does not incur any cost for running it.");
```
Comment (brian-harris) [inline extensions/sql-migration/src/constants/strings.ts:183]: I think DMS should not be abbreviated but spelled out as "Database Migration Service" There may be several instances of this to address.

File: extensions/sql-migration/src/constants/strings.ts [hunk 3]
```diff
@@ -180,32 +179,32 @@ export const SELECT_BLOB_CONTAINER = localize('sql.migration.blob.container.sele
...
+export const AUTHENTICATION_KEYS = localize('sql.migration.authentication.types', "Authentication keys");
 export function SQL_MIGRATION_SERVICE_DETAILS_HEADER(sqlMigrationServiceName: string) {
 	return localize('sql.migration.service.header', "Azure Database Migration Service \"{0}\" details:`", sqlMigrationServiceName);
 }
-export const DMS_PORTAL_INFO = localize('sql.migration.dms.portal.info', "Please note that any existing Azure Database Migration Service (DMS) in Azure portal will not show up in Azure Data Studio. DMS created in Azure Data Studio will not be visible in Azure portal yet.");
+export const DMS_PORTAL_INFO = localize('sql.migration.dms.portal.info', "Any existing Azure Data Migration Services (DMS) in the Azure portal do not appear in Azure Data Studio. Any DMS created in Azure Data Studio will not be visible in the Azure portal yet.");
 export const DATABASE_MIGRATION_SERVICE_AUTHENTICATION_KEYS = localize('sql.migration.database.migration.service.authentication.keys', "Database migration service authentication keys");
```
Comment (brian-harris) [inline extensions/sql-migration/src/constants/strings.ts:196]: * "Azure Data Migration Services" should be changed to "Azure Database Migration Service" 
* I don't recommend using "DMS" in any strings even (DMS)

File: extensions/sql-migration/src/wizard/subscriptionSelectionPage.ts [hunk 4]
```diff
@@ -1,187 +0,0 @@
-/*---------------------------------------------------------------------------------------------
- *  Copyright (c) Microsoft Corporation. All rights reserved.
- *  Licensed under the Source EULA. See License.txt in the project root for license information.
- *--------------------------------------------------------------------------------------------*/
-
-import * as azdata from 'azdata';
-import { MigrationWizardPage } from '../models/migrationWizardPage';
-import { MigrationStateModel, StateChangeEvent } from '../models/stateMachine';
-import { SUBSCRIPTION_SELECTION_PAGE_TITLE, SUBSCRIPTION_SELECTION_AZURE_ACCOUNT_TITLE, SUBSCRIPTION_SELECTION_AZURE_PRODUCT_TITLE, SUBSCRIPTION_SELECTION_AZURE_SUBSCRIPTION_TITLE } from '../constants/strings';
```
Comment (brian-harris) [inline extensions/sql-migration/src/wizard/subscriptionSelectionPage.ts:9]: are these resource strings still in use or can the be deleted as well

File: extensions/sql-migration/src/constants/strings.ts (hunk 3 above)
Comment (rchlkm) [reply extensions/sql-migration/src/constants/strings.ts]: removed all instances of DMS

File: extensions/sql-migration/src/wizard/subscriptionSelectionPage.ts (hunk 4 above)
Comment (rchlkm) [reply extensions/sql-migration/src/wizard/subscriptionSelectionPage.ts:9]: looks like I only deleted some of them in the first commit. should I remove all the unused strings?

Comment (brian-harris) [reply extensions/sql-migration/src/wizard/subscriptionSelectionPage.ts:9]: yes, that would be good, let's treat them like dead code and avoid reviewing and localizing them in the future.

File: extensions/sql-migration/src/constants/strings.ts [hunk 5]
```diff
@@ -181,32 +133,28 @@ export const SELECT_BLOB_CONTAINER = localize('sql.migration.blob.container.sele
...
+export const AUTHENTICATION_KEYS = localize('sql.migration.authentication.types', "Authentication keys");
 export function SQL_MIGRATION_SERVICE_DETAILS_HEADER(sqlMigrationServiceName: string) {
 	return localize('sql.migration.service.header', "Azure Database Migration Service \"{0}\" details:`", sqlMigrationServiceName);
 }
-export const DMS_PORTAL_INFO = localize('sql.migration.dms.portal.info', "Please note that any existing Azure Database Migration Service (DMS) in Azure portal will not show up in Azure Data Studio. DMS created in Azure Data Studio will not be visible in Azure portal yet.");
-export const DATABASE_MIGRATION_SERVICE_AUTHENTICATION_KEYS = localize('sql.migration.database.migration.service.authentication.keys', "Database migration service authentication keys");
+export const DMS_PORTAL_INFO = localize('sql.migration.dms.portal.info', "Any existing Azure Data Migration Service in the Azure portal do not appear in Azure Data Studio. Any Database Migration Service created in Azure Data Studio will not be visible in the Azure portal yet.");
```
Comment (brian-harris) [inline extensions/sql-migration/src/constants/strings.ts:145]: The service names for:
lines 139, 140:  'Data Migration Service' should be "Database Migration Service"  
lines 145:  "Azure Data Migration Service" should be "Azure Database Migration Service"

Comment (rchlkm) [reply extensions/sql-migration/src/constants/strings.ts]: thanks for catching this

**Review by brian-harris — APPROVED** [review]

**Pull request merged**